site/static/examples/kind-with-registry.sh

```diff
@@ -22,10 +22,10 @@ containerdConfigPatches:
 EOF
 
 # connect the registry to the cluster network
-docker network connect "kind" "${reg_name}"
+docker network connect "kind" "${reg_name}" 2>/dev/null || true
 
 # tell https://tilt.dev to use the registry
 # https://docs.tilt.dev/choosing_clusters.html#discovering-the-registry
 for node in $(kind get nodes); do
-  kubectl annotate node "${node}" "kind.x-k8s.io/registry=localhost:${reg_port}";
+  kubectl annotate node "${node}" "tilt.dev/registry=localhost:${reg_port}"
```

Comment:
tilt had previously agreed to use this annotation, but /shrug, there's a KEP to do this in a better way soon anyhow. #1543

Reply:
haha, sorry, we didn't get a release out fast enough with this support :facepalm: